src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -970,19 +971,19 @@ Parameter Plugin::GetMetric(const std::string& name, const std::map<std::string,
         std::vector<ov::PropertyName> cachingProperties;
         cachingProperties.push_back(ov::PropertyName(uarch_version.name(), PropertyMutability::RO));
         cachingProperties.push_back(ov::PropertyName(execution_units_count.name(), PropertyMutability::RO));
-        cachingProperties.push_back(ov::PropertyName("GPU_DRIVER_VERSION", PropertyMutability::RO));
+        cachingProperties.push_back(ov::PropertyName(driver_version.name(), PropertyMutability::RO));
         return decltype(ov::caching_properties)::value_type(cachingProperties);
-    } else if (name == "GPU_DRIVER_VERSION") {
-        return device_info.driver_version;
+    } else if (name == ov::intel_gpu::driver_version) {
+        return decltype(ov::intel_gpu::driver_version)::value_type {device_info.driver_version};
     } else if (name == ov::device::architecture) {
         std::stringstream s;
-        s << "GPU.";
+        s << "GPU: ";
         if (device_info.gfx_ver.major == 0 && device_info.gfx_ver.minor == 0) {
             s << device_info.dev_name;
         } else {
-            s << static_cast<int>(device_info.gfx_ver.major) << "."
-              << static_cast<int>(device_info.gfx_ver.minor) << "."
-              << static_cast<int>(device_info.gfx_ver.revision);
+            s << "v" << static_cast<int>(device_info.gfx_ver.major)
+              << "." << static_cast<int>(device_info.gfx_ver.minor)
+              << "." << static_cast<int>(device_info.gfx_ver.revision);
```

Comment:
sorry, but this `architecture` is used?

Reply:
Since cache blobs cannot be shared between different architectures, `ov::device::architecture` is used to generate different hash values for each architecture as below.
https://github.com/openvinotoolkit/openvino/blob/cc219d085ec5d8526b64598c918773ea5f31969a/src/inference/src/ie_core.cpp#L510-L517

Reply:
Why `uarch_version` is not used for that purpose?

Reply:
The above hash calculation logic is common across all devices. `uarch_version` is under the `ov::intel_gpu` namespace. So, `uarch_version` is added in `cachingProperties`.